tests/secrets/apis/secrets.test.ts

```diff
@@ -392,4 +392,30 @@ describe("Secrets API", () => {
     expect(result[0].id).to.be.a('string').and.satisfy((id: string) => id.startsWith(`${baseURL}/secrets/${secretName}/`));
     expect(result[0].version).to.be.a('string').and.match(MD5_REGEX);
   });
-});
\ No newline at end of file
+   it(`Should work using HTTPS endpoint @loki`, async () => {
```

Comment:
This does not actually test the fix you have implemented. The original test you created in `https.tests.ts` was perfectly suitable 😄 see below example of your original test logic and expectations:

```
const secretName: string = "initialsecret";
const secretValue: string = "initialSecretValue";

await secretClient.setSecret(secretName, secretValue);
await secretClient.setSecret("secret", "secretValue");

const result = await secretClient.getSecret("secret");

expect(result.name).to.not.equal(secretName);
expect(result.value).to.not.equal(secretValue);
```

Reply:
The test setup void the original, since the beforeeach already creates a secret containing "secret" in the name, it was getting that instead of the "initialsecret". I'll modify the names to be unique and change to not.equal anyways